18F/identity-idp
# Remove unused flags images

## 🛠 Summary of changes

Removes two unused flags images.

We use a flags sprite for the phone input component, but it is loaded directly from the `node_modules` package using the asset pipeline:

https://github.com/18F/identity-idp/blob/f00bb67d61d288bec7274b0ce25e8a2be90d68dc/config/initializers/assets.rb#L8-L10

See also:

- [Built package assets](https://unpkg.com/browse/intl-tel-input@18.3.3/build/img)
- [Code reference to sprite](https://github.com/18F/identity-idp/blob/f00bb67d61d288bec7274b0ce25e8a2be90d68dc/app/components/phone_input_component.scss#L11-L15)

The directory name is also not correct, since the package we use is `intl-tel-input`, not `intl-tel-number`.

## 📜 Testing Plan

Verify there is no regression in the appearance of flags in the phone input component:

1. Go to http://localhost:3000
2. Sign in
3. Click "Add a phone number" from your account dashboard
4. (Reauthenticate if prompted)
5. Verify the phone input shows country flags in the dropdown

## 👀 Screenshots

There are no visual changes, but here's the phone dropdown after these changes as verification that there is no regression:

![image](https://github.com/18F/identity-idp/assets/1779930/fc39cd68-3b22-4e3f-8da1-5ad7626d5598)
